grpc-client/main.go

```diff
@@ -0,0 +1,52 @@
+/*
+
+Client is just for debugging purposes, it sends the same request as crossplane would send
```

Comment:
If this file is for test/debugging purposes, I would change the name to `test` (or prefix it) and also put the whole directory under a `test` folder

Reply:
I moved it to the tests, but I'm leavinbg it as it is - it's for future us who want's to test if request is passing well